Common/util/c99_snprintf.h

```diff
@@ -0,0 +1,55 @@
+//=============================================================================
+//
+// Adventure Game Studio (AGS)
+//
+// Copyright (C) 1999-2011 Chris Jones and 2011-20xx others
+// The full list of copyright holders can be found in the Copyright.txt
+// file, which is part of this source code distribution.
+//
+// The AGS source code is provided under the Artistic License 2.0.
+// A copy of this license can be found in the file License.txt and at
+// http://www.opensource.org/licenses/artistic-license-2.0.php
+//
+//=============================================================================
+
+// borrowed and modified from [Stack Overflow](http://stackoverflow.com/q/34912145/1136311)
+
+#ifndef __AGS_CN_UTIL__C99_SNPRINTF_H
+#define __AGS_CN_UTIL__C99_SNPRINTF_H
+
+#if defined(_MSC_VER) && (_MSC_VER < 1900)
+
+#include <stdarg.h>
+#include <stddef.h>
+#include <stdio.h>
+
+#define snprintf c99_snprintf
+#define vsnprintf c99_vsnprintf
+
+__inline int c99_vsnprintf(char *outBuf, size_t size, const char *format, va_list ap)
+{
+    int count = -1;
+
+    if (size != 0)
+        count = _vsnprintf_s(outBuf, size, _TRUNCATE, format, ap);
+    if (count == -1)
+        count = _vscprintf(format, ap);
+
+    return count;
+}
+
+__inline int c99_snprintf(char *outBuf, size_t size, const char *format, ...)
+{
+    int count;
+	va_list ap;
```

Comment:
This is the only line that uses tab as indentation; really stands out, this is how I noticed it.